modules/OAuth2Clients/language/en_us.lang.php

```diff
@@ -43,7 +43,15 @@
 }
 
 $mod_strings = array (
-    'LBL_MODULE_NAME' => 'OAuth2Clients',
-    'LBL_MODULE_TITLE' => 'OAuth2Clients',
-    'LBL_MODULE_ID'=> 'OAuth2Clients',
+    'LBL_MODULE_NAME' => 'OAuth2 Clients',
```

Comment:
Just to confirm, the name will be OAuth2 not OAuth ? Even if OAuth2 is not the brand name but the brand + the version number placed here in a single word?

Reply:
Yes, so it seems. @daniel-samson